### ISSUE TITLE
Can I output the name of the test that failed?

### ISSUE THREAD

**Issue by tylerssn** [squizlabs/PHP_CodeSniffer#2537]
I've added the following to my `.phpcs.xml`

```xml
    <rule ref="Generic.Commenting.Todo">
        <message>Please review this TODO comment: %s</message>
        <severity>3</severity>
    </rule>
```

My PHP file includes:

```php
<?php
// TODO: Change the autogenerated stub
```

I want to suppress this the rule warning if my severity is high enough to warrant the suppression. I've tried various `ref` for the `Todo` rule and I've tested `vendor/bin/phpcs` at various severities and still have the output:

```log
 19 | WARNING | [ ] Comment refers to a TODO task "Change the autogenerated stub"
```

**Comment by gsherwood** [comment]
I'm not exactly sure what you are asking, so I'll provide a bit of information and see if it helps.

Firstly, you can output the name of the test that failed by using the `-s` command line argument. This prints the error code next to each error message.

Second, you've set the severity of the message to `3` and the default severity filter for PHPCS is `5`, so this message would be hidden unless you lower the severity filter of PHPCS using the `--severity` command line option. The lower the number you set, the less severe the error is.

There is a section in the wiki about using error severity: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#filtering-errors-and-warnings-based-on-severity

Does any of that information help, or do you have more questions?

**Comment by gsherwood** [comment]
Closing due to no feedback.
